server/FakeToOriginalIPConverter/package.json

```diff
@@ -5,7 +5,8 @@
   "main": "FakeToOriginalIPConverter.js",
   "scripts": {
     "prestart": "npm install",
-    "start": "node FakeToOriginalIPConverter.js"
+    "start_modify_loadfile": "node FakeToOriginalIPConverter.js --modify=load",
+    "start_modify_testsuite_config": "node FakeToOriginalIPConverter.js --modify=config"
   },
   "keywords": [
     "swagger"
```

Comment:
In the dependency , "yargs": "^17.7.2" needs to be added.

Reply:
done

Reply:
Also Prathiba, I have updated README file with the commands that can be used to modify either load.json or testsuite config file, we can convey this to Rajitha

Reply:
Thanks.